packages/clerk-js/webpack.config.js

```diff
@@ -26,17 +27,24 @@ const variantToSourceFile = {
 
 /** @type { () => import('webpack').Configuration } */
 const common = ({ mode }) => {
+  // For production releases, ignore the .env.local and read from .env.prod instead
+  const flags = mode === 'production' ? getFeatureFlags().prod : getFeatureFlags().local;
```

Comment:
❓ `mode === 'production'` includes both `@latest` and `@staging` releases, right?

Reply:
Right now yes, both staging and latest are production builds as far as webpack is concerned.

The flags will change on the env level